src/api/app/mixins/webui/manage_relationships.rb

```diff
@@ -29,6 +19,10 @@ def save_person_or_group(what)
     end
   end
 
+  def users_path
+    url_for(action: :users, project: @package ? @project : @project.name, package: @package)
```

Comment:
Passing `@project` or `@project.name` generates the same URL, so this ternary operator isn't needed.

Reply:
done

Reply:
Yeah :)

Reply:
Hmm weired doesn't show here that it is outdated

Reply:
Upps something got lost sorry